joomla/joomla-cms
# Add support for PHP-CS-Fixer

#### Summary of Changes

Adds support for the [PHP-CS-Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer) tool as a means to help programmatically review and correct code style issues in PHP code.  This initial implementation only includes the non-third-party libraries directory code.
#### Testing Instructions

Review the package's documentation to ensure the selected fixers are in line with the project's code style rules and review the changes made (sorry it's a lot of files but it's either that or do a PR for a couple directories at a time) to ensure they are consistent with the project's code style rules.

Note some changes in here are more for consistency and behavior than a PHPCS enforcement such as changing `return null;` to `return;` or the trailing comma on multi-line arrays.
